src/lib.rs

```diff
@@ -289,6 +289,9 @@ extern crate std;
 // Re-export core for use by macros
 #[doc(hidden)]
 pub extern crate core as __core;
+// Re-export memoffset for use by macros
+#[doc(hidden)]
+pub extern crate memoffset as __memoffset;
```

Comment:
Since we now re-export `offset_of`, I think this is unnecessary?

Reply:
I think we'd have to re-"export" it the old macro way for that (by having a macro that expands to the other macro)... `adapter` isn't even a public module, so the `pub use` in there actually makes little sense.

But then the wrapping macro still needs this reexport here. But I could move the `pub use` to the root and maybe that's the same thing?